src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -51,5 +64,6 @@ public double getPosition() {
   @Override
   public void periodic() {
 
+  
```

Comment:
In the periodic method (which is constantly running) we should add code to set the armMotor power based on the pid.

This would be similar to the line of code in our drive periodic where we use a PID controller https://github.com/frc1675/frc1675-2023/blob/main/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java#L240

```armMotor.set(pid.calculate(getPosition()-targetPosition);```